parts/pluginChanges.html

```diff
@@ -4,82 +4,4 @@
     <li>Fix: WMP-084 Webstorm 2020.1 NoClassDefFoundError</li>
 </ul>
 
-<p>v1.1.1</p>
-<ul>
-    <li>Fix: WMP-075 No image for Nejire Hado for Waifu of the Day</li>
-</ul>
-
-<p>v1.1.0</p>
-<ul>
-    <li>WMP-073 Properly style Waifu of the Day based on theme</li>
-    <li>WMP-069 Add settings navigation menu on Plugin toolbar menu</li>
-    <li>WMP-066 Migration to dynamic plugin</li>
-    <li>WMP-060 Support IDEA Version to 2020.1</li>
-    <li>WMP-015 Separate settings for alert</li>
-    <li>WMP-056 Support for playing different sound formats</li>
-    <li>WMP-053 Global project closing sayonara</li>
-    <li>WMP-026 Create a message bundle for WMP</li>
-    <li>WMP-037 Create Waifu of the Day provider</li>
-    <li>WMP-050 Update text 'source' to 'sauce'</li>
-    <li>WMP-051 Include more assets for waifu of the day and events
-        <ul><b>Waifu of The Day</b>
-            <li>Update Aqua image</li>
-            <li>Shinobu Kocho</li>
-            <li>Mitsuri Kanroji</li>
-            <li>Kanae Kocho</li>
-            <li>Kanao Tsuyuri</li>
-            <li>Toru Hagakure</li>
-            <li>Ochaco Uraraka</li>
-            <li>Kyoka Jiro</li>
-            <li>Nejire Hado</li>
-            <li>Eri</li>
-            <li>Himiko Toga</li>
-            <li>Ishtar</li>
-            <li>Lucy Heartfilia</li>
-            <li>Erza Scarlet</li>
-            <li>Wendy Marvell</li>
-            <li>Nao Tomori</li>
-            <li>Kotoko Iwanaga</li>
-            <li>Inori Yuzuriha</li>
-            <li>Miku Nakano</li>
-            <li>Ichika Nakano</li>
-            <li>Nino Nakano</li>
-            <li>Itsuki Nakano</li>
-            <li>Yotsuba Nakano</li>
-        </ul>
-        <br>
-        <ul><b>Alert Assets</b>
-            <li>Nyaaan</li>
-            <li>Wwwwwwaaaaaaaaaaaaaaaaaaaaah</li>
-            <li>Wwwwaaaaaooowww</li>
-            <li>OH MY GAH!</li>
-        </ul>
-    </li>
-</ul>
-
-<p>v1.0.1</p>
-<ul>
-    <li>Add more assets for Waifu of the Day</li>
-    <li>WMP-039 asset provider not properly categorizing on the next invocation</li>
-    <li>WMP-041 update images for dark theme</li>
-    <li>WMP-042 resolve sound stopping when the alert is expired</li>
-</ul>
-
-<p>v1.0</p>
-<ul>
-    <li>WMP-008 waifu of the day</li>
-    <li>WMP-022 random motivations on all alerts</li>
-    <li>WMP-027 sayonara</li>
-    <li>WMP-028 persist the consumed assets to play until all of it are consumed</li>
-    <li>WMP-029 stop playing the audio if the alert was closed</li>
-</ul>
-
-<p>v0.1.1</p>
-<ul>
-    <li>Bumped IDEA for compatibility to >= 2019.3</li>
-</ul>
-
-<p>v0.1</p>
-<ul>
-    <li>Initial EAP Release for IDEA >= 2019.2.3</li>
-</ul>
+<a href="https://github.com/zd-zero/waifu-motivator-plugin/blob/master/CHANGELOG.md">View Changelog</a>
```

Comment:
I'm thinking if we can remove this file and use the CHANGELOG to the gradle change file
https://github.com/zd-zero/waifu-motivator-plugin/blob/bde25b786129bec54a4f1d18ab7e2bfb63113b4b/gradle.properties#L9

for it was used in build.gradle
https://github.com/zd-zero/waifu-motivator-plugin/blob/bde25b786129bec54a4f1d18ab7e2bfb63113b4b/build.gradle#L57

I don't know if Jetbrains Marketplace will properly read the markdown file for the page.

Reply:
So what we can do is just keep the markdown and then convert it to HTML during the build process.

I've done that before. You can take a look at this PR if you want to https://github.com/one-dark/jetbrains-one-dark-theme/pull/157/files

Reply:
> use the CHANGELOG

Personally, I would keep the change notes concise to the new things in the latest version. Then keep the link to the changelog if they want to know all the changes.

I have not come up with anything better other than keeping a Changelog and a [release notes](https://github.com/doki-theme/doki-theme-jetbrains/blob/master/changelog/RELEASE-NOTES.md).